src/main/java/controller/LoginController.java

```diff
@@ -0,0 +1,38 @@
+package controller;
+
+import db.DataBase;
+import model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import webserver.*;
+
+public class LoginController implements Controller {
+    private static final Logger logger = LoggerFactory.getLogger(LoginController.class);
+
+    public Response serving(Request request) {
+        User user = getUserFromRequest(request);
+
+        User userById = DataBase.findUserById(user.getUserId());
+
+        if (userById == null || !userById.getPassword().equals(user.getPassword())) {
```

Comment:
getter로 값을 꺼내기 보단 객체에게 물어보는게 어떨까요?
```suggestion
        if (userById == null || !userById.checkPassword(user.getPassword())) {
```